### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by h2kfl**
add contributors and ground rule sections

### CODE REVIEW

**Review by Sungju-Huh — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
-# BugKillers
\ No newline at end of file
+# BugKillers
+
+# Contributors
+@lkwh12
+
+# Ground rules
```
Comment (yh0203kim) [inline README.md:6]: Ground Rule 추가 하여 앞으로 잘 지켜질 수 있도록 함

**Comment by minje46** [comment]
Approved

**Review by yh0203kim — APPROVED** [review]

**Comment by lkwh12** [comment]
Approved

**Comment by minje46** [comment]
check

**Review by Sungju-Huh — APPROVED** [review]
Looks good to me 👍

**Review by minje46 — APPROVED** [review]

**Review by lkwh12 — APPROVED** [review]
Approve

**Review by lkwh12 — APPROVED** [review]
Approve

**Review by yh0203kim — APPROVED** [review]
Approve

**Review by yh0203kim — APPROVED** [review]

**Review by yh0203kim — APPROVED** [review]

**Review by yh0203kim — APPROVED** [review]

**Review by yh0203kim — APPROVED** [review]

**Review by yh0203kim — APPROVED** [review]

**Pull request merged**